modules/integration/tests-integration/tests-backend/src/test/resources/testng.xml

```diff
@@ -75,4 +75,10 @@
             <class name="org.wso2.am.integration.tests.prototype.APIM23VisibilityOfPrototypedAPIInStoreTestCase"/>
         </classes>
     </test>
+
+    <test name="apim-integration-tests-token" preserve-order="true" parallel="false">
```

Comment:
Let's have it in the old place itself. If we are to break these into packages there will be many test tags

Reply:
fixed in bbb8035